packages/rum-core/src/domain/rumEventsCollection/resource/resourceCollection.ts

```diff
@@ -224,3 +225,16 @@ function computePageStateInfo(pageStateHistory: PageStateHistory, startClocks: C
     },
   }
 }
+
+function computeRequestDuration(pageStateHistory: PageStateHistory, startClocks: ClocksState, duration: Duration) {
+  // TODO remove FF in next major
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Do we need to wait for a major?

Reply:
We could have some customers that rely on before_send and expect resource.duration to be defined